### ISSUE TITLE
Configure Medicare connection for IdG in prod

### ISSUE THREAD

**Issue by thekaveman** [cal-itp/benefits#2503]
### Configuration changes
- [x] Configure a new `ClaimsProvider`
   - [x] Similar to the one set up in [#2364](https://github.com/cal-itp/benefits/issues/2364), except use the production IdG 
   - [x] Make sure the name is `cdt-medicaregov` per [message in Slack on 2024-10-10](https://cal-itp.slack.com/archives/C037Y3UE71P/p1728577692289999) (the name is seen in analytic events)

**Comment by angela-tran** [comment]
Sorry I think I might've gotten ahead of ourselves -- I didn't read the title too closely and thought this ticket represented setting up a Medicare flow, but now I'm realizing this might've just been to set up a `ClaimsProvider`. 

Since I already did the steps to configure a Medicare flow in prod, or at least the parts aren't specific to a transit agency, I documented those steps in separate ticket #2510. The flow is not linked to any agency and will not show up on the rider-facing side of the app.
